GUI/QtQuick/qml/Settings/ModifyArmoryServer.qml

```diff
@@ -0,0 +1,202 @@
+        id: title
+        Layout.alignment: Qt.AlignTop | Qt.AlignHCenter
+        Layout.preferredHeight : title.height
+        text: qsTr("Modify custom server")
+    }
+
+    RowLayout {
+        id: row
+        spacing: 12
+
+        Layout.alignment: Qt.AlignTop | Qt.AlignHCenter
+        Layout.topMargin: 24
+        Layout.preferredHeight: 19
+
+        Label {
+            Layout.fillWidth: true
+        }
+
+        Label {
+            id: radbut_text
+
+            text: qsTr("Network type:")
+
+            Layout.leftMargin: 25
+            Layout.alignment: Qt.AlignVCenter
+
+            width: 126
+            height: 19
+
+            color: "#E2E7FF"
+            font.pixelSize: 16
+            font.family: "Roboto"
+            font.weight: Font.Normal
+        }
+
+        CustomRadioButton {
+            id: radbut_main
+
+            Layout.alignment: Qt.AlignVCenter
+
+            text: "MainNet"
+
+            spacing: 6
+            font.pixelSize: 13
+            font.family: "Roboto"
+            font.weight: Font.Normal
+
+            checked: true
+
+            // netType==0 => MainNet, netType==1 => TestNet
+            onClicked : {
+                armoryServersModel.setData(armoryServersModel.index(server_index, 0)
+                                           , 0, ArmoryServersModel.NetTypeRole)
+            }
+        }
+
+        CustomRadioButton {
+            id: radbut_test
+
+            Layout.alignment: Qt.AlignVCenter
+
+            text: "TestNet"
+
+            spacing: 6
+            font.pixelSize: 13
+            font.family: "Roboto"
+            font.weight: Font.Normal
+
+            checked: false
+
+            // netType==0 => MainNet, netType==1 => TestNet
+            onClicked : {
+                armoryServersModel.setData(armoryServersModel.index(server_index, 0)
+                                           , 1, ArmoryServersModel.NetTypeRole)
+            }
+        }
+
+        Label {
+            Layout.fillWidth: true
+        }
+    }
+
+    CustomTextInput {
+        id: name_text_input
+
+        Layout.alignment: Qt.AlignTop | Qt.AlignHCenter
+        Layout.preferredHeight : 70
+        Layout.preferredWidth: 532
+        Layout.topMargin: 10
+
+        title_text: qsTr("Name")
+
+        onTextEdited: {
+            armoryServersModel.setData(armoryServersModel.index(server_index, 0)
+                                       , name_text_input.input_text, ArmoryServersModel.NameRole)
+        }
+    }
+
+    CustomTextInput {
+        id: ip_dns_text_input
+
+        Layout.alignment: Qt.AlignTop | Qt.AlignHCenter
+        Layout.preferredHeight : 70
+        Layout.preferredWidth: 532
+        Layout.topMargin: 10
+
+        title_text: qsTr("IP/DNS")
+
+        input_validator: RegExpValidator { regExp: /(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}(,(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3})*/ }
```

Comment:
Will this validator accept host name, or only IPv4 address?
What about IPv6 address?

Reply:
Only ipv4 without host

Reply:
Will beit ok? https://www.regextester.com/104038 Or maybe some other reg exp?

Reply:
Ok for IP address, but will it accept host name?

Reply:
No will not. Ok so need remove validator